# Floating IP Addresses manager (IPFloater)
# Copyright (C) 2015 - GRyCAP - Universitat Politecnica de Valencia
# 
# This program is free software: you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation, either version 3 of the License, or
# (at your option) any later version.
# 
# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU General Public License for more details.
# 
# You should have received a copy of the GNU General Public License
# along with this program.  If not, see <http://www.gnu.org/licenses/>.
--- shutdown	2016-01-13 13:44:01.103180117 +0100
+++ shutdown.ipfloater	2016-01-13 13:44:31.727179769 +0100
@@ -69,4 +69,10 @@
 force_shutdown "$deploy_id" \
     "virsh --connect $LIBVIRT_URI destroy $deploy_id"
 
+#-------- code snip to detach floating IPs --------------------------
+source $(dirname $0)/ipfloater
+detach_ips "$deploy_id"
+# -------------------------------------------------------------------
+
 sleep 4
+
